cmd/project/project.go

```diff
@@ -35,16 +35,16 @@ func NewProjectCmd(rootOptions *cmd.RootOptions) *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:     "project",
-		Aliases: []string{"project", "prj"},
+		Aliases: []string{"prj"},
```

Comment:
I think this might break being able to call `spin project <sub-commands>` and makes it just `spin prj` but not sure

Reply:
The `Use:` bit handles full command name. I removed it because it was duplicated in help text.